### ISSUE TITLE
Unable to change directory of backups after it has been set

### ISSUE THREAD

**Issue by candroid-man** [streetwriters/notesnook#157]
**Describe the bug**

Once the user selects a directory for encrypted backups, there is no way to change it (both desktop and mobile). In my case, I put it in the wrong place, as I wanted it to be on my SD card. I even went as far as to delete the directory all together hoping that Notesnook would detect that the directory no longer exists, but after manually backing up my data, it just didn't create any files. Disabling and re-enabling the backups does not help. The issue persisted after rebooting the device.

**To Reproduce**

1. Go to settings
2. Click on `Backup & restore`
3. Select a directory on your device to store the backups

**Screenshots**

![Selection_014](https://user-images.githubusercontent.com/51019003/145656167-641ac952-e7ec-4248-b91f-902c1c8a9e5f.png)

**Desktop**
 - OS: EndeavourOS Atlantis
 - Browser Electron (AppImage)
 - Version v1.6.11

**Smartphone**
 - Device: Samsung Galaxy S10+
 - OS: Android 11
 - Browser: Native application
 - Version: v1.6.92

**Additional context**

If there is a way to change the directory, it is not easily seen from the `Backup & restore` menu, which is an important thing that needs to be in that menu

**Comment by thecodrr** [comment]
@candroid-man I don't think I understood what you are trying to do? Do you want to change the directory for the automatic backups? I am trying it on my end (Linux Electron app) and it is working as normal. You just have to enable automatic backups (in settings) to see the option to change the default storage directory.

**Comment by candroid-man** [comment]
@thecodrr Try it on the mobile version, I didn't actually test it on the desktop version because I assumed that since there was no button to do it, it wouldn't be on either platform. Sorry for the confusion.

**Comment by ammarahm-ed** [comment]
> @thecodrr Try it on the mobile version, I didn't actually test it on the desktop version because I assumed that since there was no button to do it, it wouldn't be on either platform. Sorry for the confusion.

I think there should an option to change backups directory on mobile. This is a feature request.

**Comment by candroid-man** [comment]
@thecodrr After trying it on the Linux Electron app, the `Change backups storage location` button does nothing.

**Comment by thecodrr** [comment]
> @thecodrr After trying it on the Linux Electron app, the `Change backups storage location` button does nothing. The backups are working correctly however.

Can you try running the app via Terminal to see if there is an issue popping up? Because it is working fine on my end (it should open Directory Selection dialog).

**Comment by candroid-man** [comment]
@thecodrr Is that possible using the AppImage?

**Comment by thecodrr** [comment]
> @thecodrr Is that possible using the AppImage?

Yes

**Comment by candroid-man** [comment]
@thecodrr Sure enough, there is an error:

```
Checking for update
(node:168868) ExperimentalWarning: buffer.Blob is an experimental feature. This feature could change at any time
(Use `Notesnook --trace-warnings ...` to show where the warning was created)
Update for version 1.6.11 is not available (latest version: 1.6.11, downgrade is disallowed).
[168868:1211/013950.054250:ERROR:browser_main_loop.cc(269)] Gtk: gtk_native_dialog_run: assertion '!priv->visible' failed
```

**Comment by thecodrr** [comment]
```
[168868:1211/013950.054250:ERROR:browser_main_loop.cc(269)] Gtk: gtk_native_dialog_run: assertion '!priv->visible' failed
```
This is the error. Seems to be [an issue with electron](https://github.com/electron/electron/issues/31152) with a potential fix mentioned [here](https://github.com/Watson1978/jasper/commit/fa7abea0a5677629cfe9b423191156bc72ae7109)
